ode-wrapper/src/main/java/com/trihydro/odewrapper/controller/WydotTimBaseController.java

```diff
@@ -658,25 +701,28 @@ protected void expireReduceCreateSendTims(WydotTim wydotTim, TimType timType, St
             utility.logWithDate("Found less than 2 mileposts, unable to generate TIM.");
             return;
         }
-        Milepost firstPoint = milepostsAll.get(0);
-        Milepost secondPoint = milepostsAll.get(1);
 
-        var anchor = getAnchorPoint(firstPoint, secondPoint);
+        var anchor = attemptToCalculateAnchorPoint(wydotTim.getClientId(), milepostsAll);
+        if (anchor == null) {
+            return;
+        }

```

Comment:
This section of the code now calls the attemptToCalculateAnchorPoint method which is capable of handling the IdenticalPointsException.